packages/realm-server/tests/pg-test.ts

```diff
@@ -0,0 +1,67 @@
+import { module, test } from 'qunit';
+import { prepareTestDB } from './helpers';
+import PgAdapter from '../pg-adapter';
+
+module('Postgres', function (hooks) {
+  hooks.beforeEach(async function () {
+    prepareTestDB();
+  });
+
+  test('it can connect to the DB and run migrations', async function (assert) {
```

Comment:
Maybe I am missing something but I don't see how this test exercises migrations?

Reply:
Oh I see, the migrations are run in the setup and then this test checks whether the migration succeeded.

Reply:
Are we planning on adding support for having a schema and a seed file, and use that for setting up the test db, as opposed to running all the migrations on setup?

Reply:
Yeah, the fact we even have a table with columns is because migrations ran. 

Yes there will be a schema file that is generated from the db. That’s the next ticket that I’m working on 😉